src/dotnet/ToolPackage/ToolPackageInstance.cs

```diff
@@ -184,9 +187,11 @@ private IReadOnlyList<FilePath> GetPackagedShims()
             }
         }
 
-        private ToolConfiguration DeserializeToolConfiguration(string ToolSettingsFileName, LockFileTargetLibrary library)
+        private ToolConfiguration DeserializeToolConfiguration(
+            string toolSettingsFileName,
```

Comment:
Can we get rid of this argument all together?  The settings file name is a constant in this class already.

Reply:
fixed as suggested